crates/optimism/chainspec/src/lib.rs

```diff
@@ -473,15 +606,17 @@ mod tests {
 
     #[test]
     fn latest_base_mainnet_fork_id() {
+        let base_mainnet = OpChainSpecBuilder::base_mainnet().build();
         assert_eq!(
             ForkId { hash: ForkHash([0xbc, 0x38, 0xf9, 0xca]), next: 0 },
-            BASE_MAINNET.latest_fork_id()
+            base_mainnet.latest_fork_id()
         )
     }
```

Comment:
I'd like to keep this test for the const but duplicating this for the builder as well